client/malicious.js

```diff
@@ -0,0 +1,21 @@
+import ReputationMiningClient from "./main";
+
```

Comment:
Isn't this file rather a test file?

Reply:
Moved to a newly created `./client/test/` directory.